mongodbatlas/resource_mongodbatlas_custom_db_role_test.go

```diff
@@ -70,7 +69,7 @@ func TestAccResourceMongoDBAtlasCustomDBRoles_WithInheritedRoles(t *testing.T) {
 
 	inheritRole := []matlas.CustomDBRole{
 		{
-			RoleName: fmt.Sprintf("a_test-acc-INHERITED_ROLE-%s", acctest.RandString(5)),
+			RoleName: fmt.Sprintf("test-acc-INHERITED_ROLE-%s", acctest.RandString(5)),
```

Comment:
Why is this change necessary?

Reply:
well reviewing the acceptance test I checked this, I think is better write the test correctly instead of this